bgpd/bgp_bmp.h

```diff
@@ -54,7 +54,7 @@
  */
 
 PREDECL_DLIST(bmp_qlist);
-PREDECL_HASH(bmp_qhash);
+PREDECL_RBTREE_UNIQ(bmp_qhash);
```

Comment:
heh - admire the efficiency of not changing the labels/variables/function names, but having things say "hash" when they're not seems a little confusing. maybe if this is succeeding/passing, we could have a rename pass?

Reply:
I don't know yet.  Having the person who reported this test it.  I think it will.  I gave them a test patch that changed the hash_shrink decision point to be a number of items in the hash -vs- current max to divide by 10 instead of 2.  That made a difference in convergence times..